left_arm:
  projection_evaluator: joints(left_s0,left_s1)
  longest_valid_segment_fraction: 0.05
right_arm:
  projection_evaluator: joints(right_s0,right_s1)
  longest_valid_segment_fraction: 0.05
both_arms:
  projection_evaluator: joints(right_s0,right_s1)
  longest_valid_segment_fraction: 0.05
